src/LodeRunner.Core/CommandLine/LRCommandLine.cs

```diff
@@ -229,9 +229,9 @@ private static string ValidateCommandModeDependencies(CommandResult result)
                 // What happens is that when a integer type parameter such as duration is not provided and we try to get the value utilizing method GetValueOrDefault. it throws an exception.
             }
 
-            if (duration != null && duration > 0 && !runLoop)
+            if (duration != null && duration == 0 && runLoop)
             {
-                msg += $"{SystemConstants.CmdLineValidationDurationAndLoopMessage}\n";
+                Console.WriteLine(SystemConstants.CmdLineNoticeDurationValueIgnoredMessage);
```

Comment:
Was the message output intentional change to Console?

Reply:
Sort of, it's not a direct replacement. I removed the message so that we wouldn't raise an error if the duration is set but run-loop isn't. Then, I added a different message, just to log that the duration of 0 will be ignored